formulas/hostname/templates/set-hostname.mo

```diff
@@ -84,11 +84,22 @@ wickDebug "Setting hostname to $full"
 # Update hostname
 #
 reload=false
+{{#SH_UBUNTU}}
```

Comment:
Instead of using OS detection, can we do file detection?

    if [[ -f /etc/hostname ]]; then
        # your code for updating hostname
    fi

    if [[ -f /etc/sysconfig/network ]]; then
        # existing code to update this file
    fi